Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -18,26 +18,32 @@ class BTPayPalNativeTokenizationClient {
 
     func tokenize(
         request: BTPayPalRequest,
-        returnURL: String,
+        returnURL: String?,
```

Comment:
Take it or leave it comment - since this function can't do it's job without a `returnURL` it might be more readable to move the returnURL unwrap (+ potential error throw) to the callsite of this function.

Reply:
I would like to keep it as is, It will be harder to unit test with that change.